packages/amplify-e2e-core/src/init/non-interactive-init.ts

```diff
@@ -48,6 +48,7 @@ export const nonInteractiveInitWithForcePushAttach = async (
       awscloudformation: awsProviderConfig,
     }),
     '--forcePush',
+    '--debug',
```

Comment:
should this debug stay ?

Reply:
I think it can. We run other e2e commands with the flag to help with debugging issues more quickly